UnityProject/Assets/Scripts/Systems/Clearance/Clearance.cs

```diff
@@ -1,7 +1,11 @@
 ﻿namespace Systems.Clearance
 {
+	/// <summary>
+	/// TODO: ORGANISE THESE FROM LOWEST TO HIGHEST NUMBER BASED ON CLEARANCE LEVEL.
```

Comment:
ah just noticed this comment. Yeah, it isn't a good idea to shuffle the numeric values. Literally everything will break if you do it.

Reply:
yeah that's why I left a todo for now, we can deal with these later.

Reply:
is it even a thing in ss13? Can they compare access like that?

Reply:
I mean.. It should? Honestly, I'm not sure.

Reply:
We still need to distinguish between more important levels of clearance so this is a thing that we'll need to work on at some point

Reply:
if it were part of the requirements, I would have designed a different system for clearance to make it comparable. What do we need to distinguish levels of clearance, as you put it, for? When do we need it?

Reply:
My suggestion for a new system would honestly involve reworking the entire clearance system again, not just making a new one. Clearance should be divided into groups, the higher the group in level the more access you are given to around the station rather than giving crew members individual access to each and every door.
so lets say group 00 doesn't let you access anything, group 01 lets you access public areas on the station, group 02 lets you access service job areas, group 03 lets you access medical areas, group 04 lets you access science areas, and the list goes on.
"but how are you going to prevent people from different jobs not access other people's workplaces just because they have a higher clearance?" Answer, we shouldn't. In the higher your clearance, the easier it should for you to access other areas to do your job or take over someone else's shift if a department is lacking manpower. 

This would solve our problem of not knowing when someone has a higher clearance than another player from a gameplay perspective, and we can further push this forward around many different mechanics.

From a gameplay perspective, this is a change that must be done due to how lowpop our servers are and it's easier to comprehend for newer players. From a technical perspective, it's easier to work with just numbers to assign access to people rather than long list of enums.

Reply:
interesting suggestion then but it has nothing to do in this PR. We should discuss it further in discord.

Reply:
Technically it does, it's just a bigger suggestion than just re-ordering the enum around then fixing all the prefabs.

Reply:
Both time-consuming still.

Reply:
remove from the PR for now.

Reply:
the comment?

Reply:
the method as well. It is inventing a new system we don't have. To fix the issue, it should communicate the door the same credentials that were presented to the display at the moment of interaction.

Reply:
hardcoded security clearance into this until we figure out something else.